playbooks/roles/airship-setup-deployer/tasks/main.yml

```diff
@@ -5,6 +5,15 @@
 - name: Load vip from user variables
   include_vars: "{{ socok8s_extravars }}"
 
+# In some cases, people might add iso manually.
+# Let's prevent adding the same product twice.
+- name: Check if socok8s product already present
+  shell: |
+    set -o pipefail
+    zypper products | grep -q openSUSE-Addon-socok8s
```

Comment:
Should we base it off the URL or something else in case it was added manually under a different name?

Reply:
I'm explicitly checking for a product name which is defined on our side and cannot be changed by end user.

Reply:
That is a good check then.

I would change that task though:
```
shell: zypper products | grep -q openSUSE-Addon-socok8s
register: product_present
changed_when: "product_present.rc == 1"
failed_when: "product_present.rc not in [0,1]"
```

This would allow to know if things need changing in an ansible way (if this task is changed, trigger another task).

In that case, the addition of the other task's conditional is:
```
when: product_present is changed
```